conf/modules/extra_dl.xml

```diff
@@ -10,6 +10,7 @@
     <file name="extra_pprz_dl.h"/>
   </header>
   <init fun="extra_pprz_dl_init()"/>
+  <periodic fun="extra_pprz_dl_periodic()" freq="PERIODIC_FREQUENCY" autorun="TRUE"/>
```

Comment:
can't you use TELEMETRY_FREQUENCY and add in your airframe file for the nps target `<configure name="TELEMETRY_FREQUENCY" value="512"/>`?